docs/section19/19-4.md

```diff
@@ -0,0 +1,90 @@
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None    
+
+**Evaluation Context:** Each Space  
+
+**Applicability Checks:**  
+1. Applies to residential dwelling unit spaces only.  
+  
+   
+**Function Calls:**  
+1. get_component_by_id()  I think this is supposed to return the object associated with an ID?
+2. get_most_used_weekday_hourly_schedule()  To be developed by RCT team. I created a shell for this so they know what we need.  
+
+
+
+## Rule Logic:   
+**Applicability Check 1 - Check if the space is a residential dwelling unit**  
+- For each space in the B_RMI: `for space in B_RMI...Space:`   
+    - Check if the lighting space type is of a residential dwelling unit type, if yes carry on, if not then outcome for this space is NOT_APPLICABLE: `if space.lighting_space_type in ["DWELLING_UNIT"] or space._ventilation_space_type in ["TRANSIENT_RESIDENTIAL_DWELLING_UNIT"]:`  
+        - Get the zone that the space is part of start by looping through zones: `for zone in B_RMI.Zone:`  
+            - Check if the space is associated with the zone: `if space in zone.spaces: applicable_zone = zone`  
+        
+        Infiltration schedule check
+        - Reset inf_pass_cooling boolean variable: `inf_pass_cooling = true` 
+        - Get the infiltration object associated with the zone: `infiltration_obj = applicable_zone.infiltration`  
+        - Get the multiplier schedule: `multiplier_sch = get_component_by_id(B_RMI,infiltration_obj.multiplier_schedule)`  
+        - Get the design_cooling_multiplier_schedule: `design_cooling_multiplier_sch = multiplier_sch.cooling_design_day_sequence`  
+        - Get the get_most_used_weekday_hourly_schedule (this will be a list with 24 values for each hour of a day): `most_used_weekday_hourly_schedule = get_most_used_weekday_hourly_schedule(B_RMI, ASHRAE229,multiplier_sch)`  
+        - Reset y = 0: `y = 0`
+        - Check if each value in the design_cooling_multiplier_sch matches the corresponding hourly value in the get_most_used_weekday_hourly_schedule: `for x in list(design_cooling_multiplier_sch):`
+            - Check if the hourly values are equal between the design_cooling_multiplier_sch and the get_most_used_weekday_hourly_schedule : `if most_used_weekday_hourly_schedule[y] != x: inf_pass_cooling = false`  
+            - Add 1 to y: `y = y +1`  
+
+        Occupany schedule check  
+        - Reset occ_pass_cooling boolean variable: `occ_pass_cooling = true` 
+        - Get the multiplier schedule: `multiplier_sch = get_component_by_id(B_RMI,space.occupant_multiplier_schedule)`    
+        - Get the design_cooling_multiplier_schedule: `design_cooling_multiplier_sch = multiplier_sch.cooling_design_day_sequence`  
+        - Get the get_most_used_weekday_hourly_schedule (this will be a list with 24 values for each hour of a day): `most_used_weekday_hourly_schedule = get_most_used_weekday_hourly_schedule(B_RMI, ASHRAE229,multiplier_sch)`  
+        - Reset y = 0: `y = 0`
```

Comment:
Aren't you overwriting U set in line 41? Perhaps I am missing the logic here...

Reply:
Discussed on call. Question resolved.